Probesys/bileto
# new: Allow to define a default user role

## Related issue(s)



https://github.com/Probesys/bileto/issues/646

## How to test manually



1. Reset the database with `make db-reset FORCE=true`
3. Configure an email account (https://github.com/Probesys/bileto/blob/main/docs/developers/greenmail.md#with-thunderbird-or-any-external-mail-client) with an email that doesn't exist in the database yet (e.g. `dominique@example.com` )
4. Send an email with this email to `support@example.com` and fetch the mailboxes
6. Verify that a new ticket is opened by `dominique@example.com`  and that the user now exists in the database

## Checklist
